CHANGELOG.md

```diff
@@ -7,11 +7,17 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- From `currentDensity` to `density` on `EXPERIMENTAL_Table`
+
 ## [9.112.15] - 2020-03-10
 
 ### Fixed
 
-- Add `useCallback` in the `showAllColumns` and `hideAllColumns` in the `EXPERIMENTAL_useTableVisibility`.
+- Add `useCallback` in the `sh
+
+wAllColumns`and`hideAllColumns`in the`EXPERIMENTAL_useTableVisibility`.
```

Comment:
```suggestion
- Add `useCallback` in the `showAllColumns` and `hideAllColumns`in the`EXPERIMENTAL_useTableVisibility`.
```
I don't know if it was github or it was really kind of broken

Reply:
Fixed